doc/dlt_for_developers.md

```diff
@@ -78,7 +119,7 @@ ${DLT_INCLUDE_DIRS}
 and to TARGET\_LINK\_LIBRARIES:
 
 ```
-${DLT_LIBRARIES}
+${DLT_LINK_LIBRARIES}
```

Comment:
True, it's only available since 3.12.
The original text (about only using ${DLT_LIBRARIES}) is also incorrect, however, as that wouldn't work when libdlt is installed in a non-standard location.

I've added some text to clarify the options one has with different CMake versions.